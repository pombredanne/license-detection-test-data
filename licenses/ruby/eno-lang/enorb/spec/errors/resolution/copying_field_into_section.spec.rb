input = <<~DOC.strip
original: value

# copy < original
DOC

describe Eno::Errors::Resolution do
  describe '::copying_field_into_section' do
    let(:error) do
      intercept_parse_error { Eno.parse(input) }
    end

    it 'provides the correct message' do
      expect(error.message).to match_snapshot
    end

    it 'provides correct selection metadata' do
      expect(error.selection).to match_snapshot
    end
  end
end
