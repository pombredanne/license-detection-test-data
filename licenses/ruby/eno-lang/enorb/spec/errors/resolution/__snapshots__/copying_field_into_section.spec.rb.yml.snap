---
"./spec/errors/resolution/copying_field_into_section.spec.rb[1:1:1]": "In line 3 a
  field is copied into a section.\n\n   Line | Content\n      1 | original: value\n
  \     2 | \n >    3 | # copy < original\n"
"./spec/errors/resolution/copying_field_into_section.spec.rb[1:1:2]":
- - 2
  - 0
- - 2
  - 17
